src/main/java/org/jabref/logic/msbib/MSBibMapping.java

```diff
@@ -24,6 +24,45 @@ public class MSBibMapping {
 
     private static final HashBiMap<Field, String> BIBLATEX_TO_MS_BIB = HashBiMap.create();
 
+    private static final HashBiMap<String, Integer> LANG_TO_LCID = HashBiMap.create();
```

Comment:
Do you really need `HashBiMap` here? Isn't BiMap enough?

See https://github.com/HugoMatilla/Effective-JAVA-Summary#52-refer-to-objects-by-their-interface for a longer explanation

Reply:
Doesnt the same logic apply for BIBLATEX_TO_MS_BIB as well?

Reply:
I think so.